app/tasks.py

```diff
@@ -0,0 +1,34 @@
+from app import celeryapp
+from app.base.models import User, Property
+
+celery = celeryapp.celery
```

Comment:
once you move the initializations from package level to module level from `celeryapp` to `celery.py`

you can have the initial driver variable defined with capitalization as `Celery`